From 2eaa4d2ccade0f4814ff4051c83fcf62bb1cdb18 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 24 Mar 2017 11:13:17 +0800
Subject: [PATCH] arm: dts: rk3288-evb: fix regulator 'No configuration' error

SYR827/SYR828 regulators complain "No configuration", which caused by
lacking of regulator-state-mem.

Change-Id: Id113c05abb8fb095bf09979b5eb8ea1c4891037b
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-act8846.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-act8846.dts b/arch/arm/boot/dts/rk3288-evb-act8846.dts
index ddbe4d024b73..7ac1a78493f4 100644
--- a/arch/arm/boot/dts/rk3288-evb-act8846.dts
+++ b/arch/arm/boot/dts/rk3288-evb-act8846.dts
@@ -105,6 +105,9 @@
 		regulator-enable-ramp-delay = <300>;
 		regulator-ramp-delay = <8000>;
 		vin-supply = <&vcc_sys>;
+		regulator-state-mem {
+			regulator-off-in-suspend;
+		};
 	};
 
 	vdd_gpu: syr828@41 {
@@ -117,6 +120,9 @@
 		regulator-always-on;
 		regulator-ramp-delay = <6000>;
 		vin-supply = <&vcc_sys>;
+		regulator-state-mem {
+			regulator-off-in-suspend;
+		};
 	};
 
 	hym8563: hym8563@51 {
-- 
2.35.3

